app/models/Users.scala

```diff
@@ -60,6 +60,10 @@ object Users {
     findSingleUser { user => user.email === email }
   }
 
+  def findById(id: Int): Option[User] = {
+    findSingleUser { user => user.id === id }
+  }
```

Comment:
If I understand Slick (the 'functional relational mapping' library we're using) correctly, then this actually results in a nice query to the DB with a 'where user.id=<id>' added to it. The filtering actually becomes the Where clause. So this is actually a good way to do it.